### DISCUSSION TITLE
Add visual highlighting a large file in size when DC is doing overwriting

### DISCUSSION THREAD

**Discussion by Aliaksandr3by** [doublecmd/doublecmd#1317] [Suggestions]
Add visual highlighting a large file in size
maybe more file size, more font size

![image](https://github.com/doublecmd/doublecmd/assets/11432724/d3b17141-3e91-4c64-870e-2525eb14bb2e)

**Comment by Alexey-T** [comment]
What does need the change here? Font size of label? it looks OK.

**Comment by alexx2000** [comment]
Font size of 'file size'. This dialog use universal function so I can't do it.

**Reply by Aliaksandr3by** [reply]
it is only my proposal, it might be a bad idea

might you add the difference between the two size of files ?
or add buttons for overwriting by size?

![image](https://github.com/doublecmd/doublecmd/assets/11432724/83840b7c-6d3c-4238-81ca-854b4a24b2e0)

**Comment by TheNicker** [comment]
I think the idea here is to add visual cue, to easily distinguish between the two files.
Different font sIzes would probably look bad.
If possible I would suggest make the text bold or colorful only when theres a difference, in the example above it is just the timestamp.

Possible Implementation:
Meta text tags could be added to the text of the dialog boxes, ignored by default. 
These could contain text formatting much like HTML

It it's a universal function, then it supposed to be a single point of change
